elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -224,8 +225,11 @@ protected ElideResponse handleRequest(boolean isReadOnly, Object opaqueUser,
         } catch (ParseCancellationException e) {
             return buildErrorResponse(new InvalidURLException(e), isVerbose);
 
-        } catch (RuntimeException | Error e) {
-            log.error("Exception uncaught by Elide", e);
+        } catch (RuntimeException e) {
```

Comment:
Probably want all exceptions. What if reflection blows up and a checked exception is thrown? We would probably ignore it?